### DISCUSSION TITLE
Bladebit ramplot

### DISCUSSION THREAD

**Discussion by prologisticsbeliani** [Chia-Network/chia-blockchain#16426] [Support]
Trying to plot K-32 compressed 7 on Chia blockchain 2.0.1, nothing happened for more than 1 hour, in the logs I see the config only
Bladebit Chia Plotter
Version      : 3.0.0
Git Commit   : 02a8e68dcf0e971c72fc8377b89cdd081b80b843
Compiled With: msvc 19.29.30151

[Global Plotting Config]
 Will create 1 plots.
 Thread count          : 56
 Warm start enabled    : false
 NUMA disabled         : false
 CPU affinity disabled : false
 Farmer public key     : [REDACTED]
 Pool contract address : [REDACTED]
 Compression Level     : 7
 Benchmark mode        : disabled

**Comment by BrandtH22** [comment]
Hey @prologisticsbeliani , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).

As for plotting what OS are you using? (there are some known bugs on windows) 

Also, are you attempting to run the plotter in the GUI or the cli? (if the cli what is the command you are using, if GUI what are the settings used)

Lastly, you will note that I have redacted the public key and pool address in your message. While making these available will not make it possible for anyone to steal your funds there are inherent privacy risks with exposing it (someone can monitor your farming activity and rewards)

**Reply by prologisticsbeliani** [reply]
Hi. I used win10 GUI. Settings you can read in 'configure', if something else you need Im ready to answer

**Reply by BrandtH22** [reply]
Hey, as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).

What is the total RAM available on your machine?

**Comment by prologisticsbeliani** [comment]
512Gb RAM

чт, 5 окт. 2023 г. в 21:25, ClydeWallace22 ***@***.***>:

**Reply by BrandtH22** [reply]
Can you try updating to the latest version and see if it is working properly for RAMplot?
https://www.chia.net/downloads/

**Comment by prologisticsbeliani** [comment]
Hi
I tried with CLI on the last version
chia plotters bladebit ramplot -n 17 -f blablba -c blabla --compress 7 -d f:\ 
 
....
 
System Memory: 501/511 GiB.
Memory required: 416 GiB.
Allocating buffers.
 
and thats all, script was finished.
 
> Can you try updating to the latest version and see if it is working properly for RAMplot?
> https://www.chia.net/downloads/
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.



-- 
С уважением,
 Алексей                         ***@***.***

**Reply by BrandtH22** [reply]
Hey @prologisticsbeliani, can you try installing and running bladebit directly from here:
NOTE - use the bladebit-cuda installer

https://github.com/Chia-Network/bladebit/releases/tag/v3.1.0

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
